bitnami/minio/values.yaml

```
@@ -1188,6 +1188,8 @@ metrics:
   ## To allow public access without authentication for prometheus metrics set environment as follows.
   ##
   prometheusAuthType: public
+  ## @param metrics.enabled Enable metrics
+  enabled: false
```

For consistency with other charts:
```suggestion
  ## @param metrics.enabled Enable the export of Prometheus metrics
  ##
  enabled: false
```